swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+
 class Item:
-    pass
+    #category is an empty string 
+    def __init__(self, category = None, condition = 0):
+        if not category:
+            category = ""
```

Comment:
Since a string is a non-mutable datatype, we do not need to do this work in the constructor, but can directly assign the default parameter `category=''`